.travis.yml

```diff
@@ -45,3 +45,12 @@ after_success:
 notifications:
   email: false
 
+jobs:
+  include:
+    - stage: Code Quality
+      php: 7.2
+      env: STATIC_ANALYSIS
+      install: travis_retry composer update --prefer-dist --prefer-stable
+      before_script:
+        - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.9
+      script: vendor/bin/phpstan analyse -l 1 src -c phpstan.neon
```

Comment:
As mentioned in the comments bellow, the level here should be changed to `0` - to start with the basic checks.